lib/module.js

```diff
@@ -6,8 +6,22 @@ module.exports = function (moduleOptions) {
     ...moduleOptions
   }
 
+  if (!options.recartSiteId) {
+    console.error('RecartSiteId is required on module options')
+    return
+  }
+  const siteId = options.recartSiteId
+  const baseUrl = options.recartBaseUrl || 'https://cdn.ghostmonitor.com'
+
   const fileName = 'recart-nacelle-plugin.js'
 
+  this.options.head.script.push({
+    hid: 'trackingjs',
+    src: `${baseUrl}/recart-loader.js?siteId=${siteId}&storefront=nacelle`,
+    async: true,
+    defer: true
```

Comment:
It's instead of, so you use async or defer. There is a main difference between them. All of them download the files in asynchronous fashion, but async loads the script right after it has been downloaded and browser won't wait until the whole page loaded. Defer, on the other hand waits until the whole page is loaded and loads / interprets the code after that.

Reply:
We usually use async, btw.

Reply:
Good to know thanks, I will use async then here too

Reply:
> When adding external / remote scripts, use the `defer` option whenever possible - this helps to reduce the performance toll of external scripts by executing the script after the `document` is parsed
>
> https://docs.getnacelle.com/nuxt/app-integrations-best-practices.html